src/main/java/com/google/devtools/build/lib/bazel/rules/android/BazelAndroidSemantics.java

```diff
@@ -36,8 +38,18 @@
 public class BazelAndroidSemantics implements AndroidSemantics {
   public static final BazelAndroidSemantics INSTANCE = new BazelAndroidSemantics();
 
-  private BazelAndroidSemantics() {
-  }
+  private static final ImmutableSet<PackageIdentifier> STARLARK_MIGRATION_NATIVE_USAGE_ALLOW_LIST =
+      // Internal package identifiers that are allowed to use the native Android rules until they
+      // can be fully moved into the rules_android Starlark implementation.
+      ImmutableSet.<PackageIdentifier>builder()
+          .add(
+              PackageIdentifier.createUnchecked(
```

Comment:
@ahumesky @ted-xie alternatively we can add load statements on `rules_android` to the build files here, but that might cause other issues for people that don't import the android rules.